tests/python/relay/aot/riscv.mk

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+AOT_ROOT ?= $(CRT_ROOT)/aot
+
+ENABLE_TVM_PLATFORM_ABORT_BACKTRACE = 0
+DMLC_CORE=$(TVM_ROOT)/3rdparty/dmlc-core
+TOOLCHAIN_PATH=$(shell dirname $(shell which riscv64-unknown-linux-gnu-gcc))/..
+
+CC = clang-16
+CXX = clang++-16
+
+TARGET_CFLAGS = --target=riscv64-unknown-linux-gnu -march=rv64gcv -static
+RUNNER = spike
+RUNNER_OPT = --isa=rv64gcv $(shell which pk)
+
+PKG_CFLAGS = ${PKG_COMPILE_OPTS} ${TARGET_CFLAGS} -O2 \
```

Comment:
Some reason why using `-O2` instead of `-O3` or `-Os`?

Reply:
No, there was no reason